_distro_map.yml

```diff
@@ -76,3 +76,13 @@ atomic-registry:
     master:
       name: 'Latest'
       dir: latest
+openshift-arho:
```

Comment:
I've switched it to `openshift-aro` everywhere in https://github.com/openshift/openshift-docs/pull/14652, if you can update throughout in this PR, too.